community/blog/2022-12 Rethinking event publishing in Arcus EventGrid v3.3/README.md

```diff
@@ -0,0 +1,89 @@
+# Rethinking event publishing in Arcus EventGrid v3.3
+The Arcus EventGrid v3.3 release brings event publishing to a whole new level. It both simplifies and enhances the whole process into something very powerful. Keep reading to find out why. 
+
+## Moving towards new Azure SDK
+The v3.3 release steps away from defining our very own event publishing contract. Previously, we used an `IEventGridPublisher` interface and builder to create an instance of this interface. This instance could be injected into all the parts of your system that needed event publishing. We used it ourselves when we needed to publish events based on potential expired client secrets. There is, however, a problem here that only recently became a huge one: Microsoft's [`EventGridPublisherClient`](https://www.nuget.org/packages/Azure.Messaging.EventGrid/) does exactly that. With the dependency injection-supported Azure clients, you could inject such a client and publish events without Arcus. This is what software is about, something new comes up and everything changes.
```

Comment:
Minor remark.
Probably on more things, so I'd say 'For instance, we used it ourselves ...'.

Reply:
Is it a problem that Microsoft's EventGridPublisherClient does exactly that ? 
I would rephrase that to something like:

> However, recently Microsoft introduced the `EventGridPublisherClient` class which does exactly the same thing.

Reply:
Ok, fixed these two thing. Thx!